kk/models/images.py

```diff
@@ -15,16 +12,16 @@ def get_images_dir():
 file_storage = FileSystemStorage(location=get_images_dir())
 
 
-class CommonImage(ModifiableModel):
+class Image(ModifiableModel):
```

Comment:
So what's the reason for changing the data model? I'm not saying it's a bad thing, I'd just like to know why.